embedded-db-junit-liquibase/src/main/java/org/zapodot/junit/db/plugin/LiquibaseInitializer.java

```diff
@@ -138,7 +138,7 @@ public LiquibaseInitializer build() {
                         "You must provide a changelog file to the LiquibaseIntitializer Plugin builder");
             }
             try {
-                if (resourceAccessor.getResourcesAsStream(databaseChangeLog) == null) {
+                if (resourceAccessor.openStream(null, databaseChangeLog) == null) {
```

Comment:
Looks like it's compatible with the way this method is used within Liquibase:

https://github.com/liquibase/liquibase/search?q=openStream